drizzlepac/mapreg.py

```diff
@@ -16,10 +16,9 @@
 from stsci.tools import teal
 from . import util
 from .regfilter import fast_filter_outer_regions
+from .version import __version__
 
 
-# This is specifically NOT intended to match the package-wide version information.
-__version__ = '0.1'
 __taskname__ = 'mapreg'
 __author__ = 'Mihai Cara'
 
```

Comment:
This module leaves __taskname__ and __author__ intact, but photeq.py removes these entries.  Are we removing all the dunders?  We should be consistent, unless there is some rationale that I am missing.